inception/inception/image_processing.py

```diff
@@ -401,7 +404,8 @@ def parse_example_proto(example_serialized):
   return features['image/encoded'], label, bbox, features['image/class/text']
 
 
-def batch_inputs(dataset, batch_size, train, num_preprocess_threads=None):
+def batch_inputs(dataset, batch_size, train, num_preprocess_threads=None,
+                 num_readers=1):
```

Comment:
They could, but usually for training we want multiple readers to increase speed and randomness, while for evaluation we want sequential reads, and usually 1 reader is enough.